client/components/Editor/Slides.jsx

```diff
@@ -226,7 +215,7 @@ class Slides extends React.Component {
                             ))}
                         </Sortable>
                     </Grid.Column>
-                    <Grid.Column width={8}>
+                    <Grid.Column width={10}>
```

Comment:
Did this change cause this?

![image](https://user-images.githubusercontent.com/27985/67689512-cc8e2880-f971-11e9-9e48-20287cc761dd.png)

Reply:
I think so? Though the added space to the right of the editor doesn't look as wide on my screen. I adjusted the columns a bit since they were so crunched previously.

Reply:
Let me know if people prefer different widths, I could split the difference and do `9`

Reply:
I feel like we have have the slide thumbnails closer to the editor. `9` seems good.